client/state/site-settings/test/actions.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useNock from 'test/helpers/use-nock';
+import {
+	SITE_SETTINGS_RECEIVE,
+	SITE_SETTINGS_REQUEST,
+	SITE_SETTINGS_REQUEST_FAILURE,
+	SITE_SETTINGS_REQUEST_SUCCESS,
+	SITE_SETTINGS_SAVE,
+	SITE_SETTINGS_SAVE_FAILURE,
+	SITE_SETTINGS_SAVE_SUCCESS,
+	SITE_SETTINGS_UPDATE
+} from 'state/action-types';
+import {
+	receiveSiteSettings,
+	requestSiteSettings,
+	saveSiteSettings,
+	updateSiteSettings
+} from '../actions';
+
+describe( 'actions', () => {
+	const spy = sinon.spy();
+
+	beforeEach( () => {
```

Comment:
You could also use the [`useSandbox` test helper](https://github.com/Automattic/wp-calypso/tree/master/test/test/helpers/use-sinon) which resets automatically between tests:

```js
let spy;
useSandbox( ( sandbox ) => spy = sandbox.spy() );
```